blueprint/abac/hybrid-client-devices.md

```diff
@@ -186,3 +186,7936 @@ This is an example AutopilotConfigurationFile.json only. This file will be agenc
+Set-RuleOption -FilePath $WDACPolicy -Option 16 # Enable No Reboot
+Set-RuleOption -FilePath $WDACPolicy -Option 19 # Enable Dynamic Code Security
+```
+7. Deploy the file locally in audit mode and validate no additional files require whitelisting
+8. Using an administrative PowerShell session run the following to switch the policy into enforce mode
+```powershell
+Set-RuleOption -FilePath $WDACPolicy -Option 3 -Delete
+```
+9. Deploy the file locally in enforce mode and validate no additional files require whitelisting
+10. [Deploy the file globally](/blueprint/abac/hybrid-client-devices.html#wdac-policy---deployment) 
+
+
+#### Predefined Microsoft Base 
+
+Microsoft have developed a number of base policies which provide the whitelisting rules for a standard Windows 10 image. These base policies are located on any Windows 10 machine under `%windir%\schemas\CodeIntegrity\ExamplePolicies`. Within that directory, the policy titled `DefaultWindows_Audit.xml` should be leveraged as the base policy. It includes the rules necessary to ensure that Windows, 3rd party hardware and software kernel drivers, and Windows Store apps will run.  
+
+The procedure to leverage this policy is as follows:
+1. Make a copy of the policy and place it into a working directory (i.e. My documents)
+2. Open the policy in a XML editor such as Visual Studio code or notepad.
+3. Remove all flighting related signers (note they will also need to be removed from the signing scenarios)
+4. Using an administrative PowerShell session run
+```powershell
+$WDACPolicy = path to policy file
+Set-RuleOption -FilePath $WDACPolicy -Option 0 # Enabled UMCI
+Set-RuleOption -FilePath $WDACPolicy -Option 1 # Enable Boot Menu Protection
+Set-RuleOption -FilePath $WDACPolicy -Option 2 # Require WHQL
+Set-RuleOption -FilePath $WDACPolicy -Option 3 # Enable Audit Mode
+Set-RuleOption -FilePath $WDACPolicy -Option 4 # Disable Flight Signing
+Set-RuleOption -FilePath $WDACPolicy -Option 6 # Enable Unsigned Policy
+Set-RuleOption -FilePath $WDACPolicy -Option 8 # Require EV Signers
+Set-RuleOption -FilePath $WDACPolicy -Option 10 # Enable Boot Audit on Failure
+Set-RuleOption -FilePath $WDACPolicy -Option 12 # Enable Enforce Store Apps
+Set-RuleOption -FilePath $WDACPolicy -Option 13 # Enable Managed Installer
+Set-RuleOption -FilePath $WDACPolicy -Option 16 # Enable No Reboot
+Set-RuleOption -FilePath $WDACPolicy -Option 17 # Enable Allow Supplemental
+Set-RuleOption -FilePath $WDACPolicy -Option 19 # Enable Dynamic Code Security
+```
+5. Deploy the file locally in audit mode and validate no additional files require whitelisting
+6. Using an administrative PowerShell session run the following to switch the policy into enforce mode
+```powershell
+Set-RuleOption -FilePath $WDACPolicy -Option 3 -Delete
+```
+7. Deploy the file locally in enforce mode and validate no additional files require whitelisting
+8. [Deploy the file globally](/blueprint/abac/client-devices.html#wdac-policy---deployment) 
```

Comment:
This links from hybrid client devices to the cloud native version. Should it be linking to the hybrid copy?

Reply:
updated.